x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -92,6 +96,15 @@ interface GetAlertParams {
   index?: string;
 }
 
+interface GetAlertSummaryParams {
+  id?: string;
+  gte: string;
+  lte: string;
+  featureIds: string[];
+  filter?: estypes.QueryDslQueryContainer[];
```

Comment:
Since the filter is an object, the type should be an object too. (`estypes.QueryDslQueryContainer`)

Reply:
Yes last night I thought mess up this one, let me check again, but I do think we want array of it.